_build/templates/default/sass/installer.scss

```diff
@@ -1,6 +1,12 @@
 @import "../node_modules/normalize-scss/sass/normalize/import-now";
 @import "colors-and-vars";
 
+$darkBlack: #343434;
+$lightGray: #F6F6F6;
+$softGray: #D6D6D6;
+$blue: #4A90E2;
+$green: #14C8A0;
```

Comment:
To keep the colours consistent across MODX it's better to change the values inside [_colors-and-vars.scss](https://github.com/modxcms/revolution/blob/3.x/_build/templates/default/sass/_colors-and-vars.scss)

Reply:
I understand this, but I say that this PR is for discussion, so that global styles are not edited if the community does not accept the changes.